src/index.js

```diff
@@ -1,8 +1,10 @@
 const shiki = require("shiki");
+const fs = require("fs");
```

Comment:
```suggestion
const fsPromises = require("fs/promises");
```
Use promise because the later function is an async function, so using `readFile` should be better than `readFileSync`